tasks/tasks.go

```diff
@@ -0,0 +1,88 @@
+package tasks
```

Comment:
Could you add a top level docstring to describe this package?

Reply:
Added.